disco_aws_automation/disco_vpc.py

```diff
@@ -249,228 +258,26 @@ def _configure_dhcp(self):
             ntp_server = self.networks[ntp_server_metanetwork].ip_by_offset(ntp_server_offset)
 
         # internal_dns server should be default, and for this reason it comes last.
-        dhcp_options = self.vpc.connection.create_dhcp_options(
-            domain_name, [internal_dns, external_dns], ntp_server
-        )
-        keep_trying(300, dhcp_options.add_tag, "Name", self.environment_name)
-        return dhcp_options
+        dhcp_configs = []
+        dhcp_configs.append({"Key": "domain-name", "Values": [domain_name]})
+        dhcp_configs.append({"Key": "domain-name-servers", "Values": [internal_dns, external_dns]})
+        dhcp_configs.append({"Key": "ntp-servers", "Values": [ntp_server]})
 
-    @staticmethod
-    def _extract_port_range(port_def):
-        ports = port_def.split(":")
-        return [int(ports[0]), int(ports[1] if len(ports) > 1 else ports[0])]
-
-    def _open_customer_ports(self):
```

Comment:
sigh. this means old disco's direct connect has to be changed before disco can easily move to asiaq.